Gems/Atom/Feature/Common/Assets/Passes/Mobile/Pipeline.pass

```diff
@@ -141,6 +155,20 @@
                         }
                     ]
                 }, 
+                {
+                    "Name": "OutlinePass",
```

Comment:
How does it composite the outline RT with the one from forward pass? How does it know which pixels are obstructed behind a wall and which ones aren't. Is it using depth test?

Reply:
The outline pass doesn't use the stencil - that happens in the OutlineGatherPass.  This one just composites the outlines onto the scene

Reply:
Oh I see. For the gather pass you are using Less than equal for depth test so that way it just gets composted on as is.